libs/dextool/source/dextool/compilation_db/package.d

```diff
@@ -739,19 +764,31 @@ ParseFlags parseFlag(CompileCommand cmd, const CompileCommandFilter flag_filter,
                 st = State.keep;
             } else if (st == State.pathArgumentToAbsolute) {
                 st = State.keep;
-                auto p = buildNormalizedPath(workdir, arg).absolutePath;
-                rval.put(p);
-                includes.put(p);
+                putNormalizedAbsolute(arg);
             } else if (st == State.priorityKeepNextArg) {
                 st = State.keep;
                 rval.put(arg);
+            } else if (st == State.checkingForQuotation) {
+                if (!path.data.empty) {
+                    path.put(" ");
+                }
+                path.put(arg);
+                if (isEndingWithQuotation(arg)) {
+                    st = State.keep;
+                    putNormalizedAbsolute(path.data.join.strip('"'));
```

Comment:
By changing the type of `path` to string this can be made more efficient by changing it to `path.data[0 .. $-1]`